x-pack/filebeat/module/rapid7/nexpose/test/generated.log-expected.json

```diff
@@ -0,0 +1,1901 @@
+[
+    {
+        "event.code": "NEXPOSE_GENERIC",
+        "event.dataset": "rapid7.nexpose",
+        "event.module": "rapid7",
+        "event.original": "%NEXPOSE-nci: SiteEventHandler deny",
+        "fileset.name": "nexpose",
+        "input.type": "log",
+        "log.offset": 0,
+        "observer.product": "Nexpose",
+        "observer.type": "Vulnerability",
+        "observer.vendor": "Rapid7",
+        "rsa.internal.messageid": "NEXPOSE_GENERIC",
+        "service.type": "rapid7",
+        "tags": [
+            "rapid7.nexpose",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "NEXPOSE_GENERIC",
+        "event.dataset": "rapid7.nexpose",
+        "event.module": "rapid7",
+        "event.original": "%NEXPOSE-iin: persistent-xss ",
+        "fileset.name": "nexpose",
+        "input.type": "log",
+        "log.offset": 36,
+        "observer.product": "Nexpose",
+        "observer.type": "Vulnerability",
+        "observer.vendor": "Rapid7",
+        "rsa.internal.messageid": "NEXPOSE_GENERIC",
+        "service.type": "rapid7",
+        "tags": [
+            "rapid7.nexpose",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "NEXPOSE_GENERIC",
+        "event.dataset": "rapid7.nexpose",
+        "event.module": "rapid7",
+        "event.original": "%NEXPOSE-tenima: Telling laboreet ",
+        "fileset.name": "nexpose",
+        "input.type": "log",
+        "log.offset": 66,
+        "observer.product": "Nexpose",
+        "observer.type": "Vulnerability",
+        "observer.vendor": "Rapid7",
+        "rsa.internal.messageid": "NEXPOSE_GENERIC",
+        "service.type": "rapid7",
+        "tags": [
+            "rapid7.nexpose",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "NEXPOSE_GENERIC",
+        "event.dataset": "rapid7.nexpose",
+        "event.module": "rapid7",
+        "event.original": "%NEXPOSE-giatq: SPIDER-XSS ",
+        "fileset.name": "nexpose",
+        "input.type": "log",
+        "log.offset": 101,
+        "observer.product": "Nexpose",
+        "observer.type": "Vulnerability",
+        "observer.vendor": "Rapid7",
+        "rsa.internal.messageid": "NEXPOSE_GENERIC",
+        "service.type": "rapid7",
+        "tags": [
+            "rapid7.nexpose",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "[Site:",
+        "event.dataset": "rapid7.nexpose",
+        "event.module": "rapid7",
+        "event.original": "%NEXPOSE-lupt: 2016-3-26T10:20:16 [xea] [Thread: qua] [Site: luptatev] com.rapid7.nexpose.datastore.connection.evictionThreadTime is not configured - returning default value admi.",
+        "fileset.name": "nexpose",
+        "input.type": "log",
+        "log.offset": 129,
+        "observer.product": "Nexpose",
+        "observer.type": "Vulnerability",
+        "observer.vendor": "Rapid7",
+        "rsa.internal.messageid": "[Site:",
```

Comment:
Is this right?, looks off with the [

Reply:
There's something off in this parser or the log generator. I will check